### PR TITLE
Add `People/User group` article

### PR DESCRIPTION
**Pull request by Walavouchey**
the badge images in `shared/group/` are made using figma with dimensions roughly matching. they're not pixel perfect because they don't have fixed widths on the website, but this should suffice for the wiki.

https://www.figma.com/file/37g7ubbPJRU6Ppd4WW3bXj/User-group-badges

### CODE REVIEW

File: wiki/People/User_group/en.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - usergroup
+---
+
+# User group
+
+*Notice: The Chat Bot and Announce user group listings are not public.*
+
+![User group badge example](img/user-group-badge.png "User group badge on a user's profile page")
+
+Listed below are **user groups**, most of which are made up of osu! community members that help maintain osu!. Most of these user groups can be recognised by their forum colours, in-game chat colours, profile titles, and/or user group badges.
```
Comment (TicClick) [inline wiki/People/User_group/en.md:12]: missed link insertion opportunities
```suggestion
Listed below are **user groups**, most of which are made up of osu! community members that help maintain osu!. Most of these user groups can be recognised by their [forum](/wiki/Community/Forum) colours, [in-game chat](/wiki/Client/Interface/Chat_console) colours, [profile titles](/wiki/Community/User_title), and/or user group badges.
```

File: wiki/People/User_group/en.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - usergroup
+---
+
+# User group
+
+*Notice: The Chat Bot and Announce user group listings are not public.*
```
Comment (TicClick) [inline wiki/People/User_group/en.md:8]: what's the difference between a `Note` and a `Notice`?

**Review by TicClick — DISMISSED** [review]

File: wiki/People/User_group/en.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/People/User_group/en.md:8]: image

should be `Note` i suppose

```suggestion
*Note: The Chat Bot and Announce user group listings are not public.*
```

File: wiki/People/User_group/img/user-group-badge.png
Comment (Walavouchey) [inline wiki/People/User_group/img/user-group-badge.png:1]: i wonder what id the secret ppy user group has

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**